File: quiz/quiz2/fizzBuzz.py
def fizzBuzz (x):
	remainder_3 = x % 3
	remainder_5 = x % 5 
	if (remainder_3 == 0 and remainder_5 == 0): 
		return "FizzBuzz"
	elif (remainder_3 == 0): 
		return "Fizz"
	elif (remainder_5 == 0): 
		return "Buzz"
	else: 
		return `x`



for num in range(1, 101):
	print(fizzBuzz(num))

File: quiz/quiz2/fizzBuzz.scala
def fizzBuzz(x:Int): String = (x % 3, x % 5) match {
	case (0, 0) => "FizzBuzz"
	case (0, _) => "Fizz"
	case (_, 0) => "Buzz"
	case (_, _) => x.toString()
}

println((1 to 100).map(fizzBuzz).mkString("\n "))
